rust/inc/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+#[cxx::bridge(namespace = "rust")]
```

Comment:
Missing copyright/license.

I'll pick off just the install-dependencies part since it requires extra work.

Reply:
ok

Reply:
It's queued now (97408078a17ae3da461e9db1a5af95e4a0952c78), please rebase once it promotes.